tests/operators/test_docker_operator.py

```diff
@@ -49,6 +49,7 @@ def test_execute(self, client_class_mock, mkdtemp_mock):
         client_mock.create_container.return_value = {'Id': 'some_id'}
         client_mock.create_host_config.return_value = host_config
         client_mock.images.return_value = []
+        client_mock.attach.return_value = ['container log']
         client_mock.logs.return_value = ['container log']
```

Comment:
Probably don't need this mock anymore?

Reply:
When self.xcom_push_flag is true, its still getting it from cli.logs. So keeping the this mock